pyfecons/tests/test_cas220101_cost_functions.py

```diff
@@ -1,100 +0,0 @@
-import unittest
```

Comment:
In the future make sure to have files moved rather than delete + add to preserve git history. Especially in active code.

Reply:
oh nice! didn't know about this